### ISSUE TITLE
Problem with spec-tools 0.7.0 and compojure-api 2.0.0-alpha19

### ISSUE THREAD

**Issue by dtenny** [metosin/spec-tools#117]
I was trying to figure out some problems I was having using compojure-api 2.0.0-alpha19 and spec-tools 0.6.1.  I thought perhaps using the spec-tools 0.7.0 might help, but when I try to build my system with this combination of libraries I get this error:

```
clojure.lang.ExceptionInfo: spec-coercion is not enabled. you most likely are missing the required deps: org.clojure/clojure 1.9+ and metosin/spec-tools. {:name :spec}, compiling:(routes.clj:50:3)
Exception in thread "main" clojure.lang.ExceptionInfo: spec-coercion is not enabled. you most likely are missing the required deps: org.clojure/clojure 1.9+ and metosin/spec-tools. {:name :spec}, compiling:(routes.clj:50:3)
```

Where routes.clj is my code.  However the error only shows up if I use spec-tools 0.7.0, there's no problem with spec-tools 0.6.1.   (rounes.clj line 50 is a `sweet/api` call).

It's possibly the case that `lein` has left something stale around, it wouldn't be the first time, but I ran the two comparative builds multiple times with the same result each time.  (0.6.1 worked, 0.7.0 did not work).

The above aside,  I like where spec-tools is going, it makes up for a number of clojure.spec ... omissions ..., thanks for building it.

**Comment by miikka** [comment]
Hi, compojure-api has not yet been updated for spec-tools 0.7.0.

**Comment by ikitommi** [comment]
fixed in `[metosin/compojure-api "2.0.0-alpha20"]`
